### DISCUSSION TITLE
Release notes?

### DISCUSSION THREAD

**Discussion by Danny-Rosenberg** [j-kaltes/Juggluco#93] [Q&A]
Hello, thanks for the great work on this project! Are there any release notes for app updates? For example, what was the reason for the major version update from 6.x.x to 7.x.x?

edit: I see they are here: https://www.juggluco.nl/Juggluco/changes.html

**Comment by j-kaltes** [comment]
I try to keep with the following version number convention:
1. An increase in major number when the update potentiality changes the behavior of the app;
2. A middle number increase when a new feature is added, but with small feature additions I forget to increase it;
3. A last number increase when it is only a bug fix.

The last update from 6 to 7 added new features: a widget and talking, but I thought that this could also change the existing behavior.
